### DISCUSSION TITLE
Format code with spork/fmt

### DISCUSSION THREAD

**Discussion by pepe** [swlkr/osprey#8] [General]
I want to ask you if you ever considered format your codes with formatted found in the spork lib? 

I can understand if not, but if you are interested, I would love to hear what you think about it.

As always, thank you very much for the great work.

**Comment by swlkr** [comment]
Yeah! I started to use it some of my private projects and I think I’ve finally come to peace with it haha

**Reply by pepe** [reply]
Great to hear. Can I format osprey then?

**Reply by swlkr** [reply]
Yeah format it!

**Reply by pepe** [reply]
Your codes your format
